backend/protzilla/methods/importing.py

```diff
@@ -172,6 +241,54 @@ class MetadataColumnAssignment(ImportingStep):
 
     output_keys = ["metadata_df", "protein_df"]
 
+    def create_form(self):
+        return Form(
+            label="Metadata column assignment",
+            input_fields=[
+                DropdownField(
+                    name="metadata_required_column",
+                    label="Missing, but required metadata columns",
+                ),
+                DropdownField(
+                    name="metadata_unknown_column",
+                    label="Existing, but unknown metadata columns",
+                )
+            ],
+        )
+
+    def modify_form(self, form, run):
+        metadata_required_column = form["metadata_required_column"]
+        metadata_unknown_column = form["metadata_unknown_column"]
+
+        metadata = run.steps.get_step_output(
+            ImportingStep, "metadata_df", include_current_step=True
+        )
+
+        if metadata is not None:
+            metadata_required_column.set_options([
+                Option(col, col)
+                for col in ["Sample", "Group", "Batch"]
+                if col not in metadata.columns
+            ])
+            if len(metadata_required_column.options) == 0:
+                metadata_required_column.set_options([
+                    Option("No required columns missing", "No required columns missing") #sollten die options hier vllt einfach leer gesetzt werden?
+                ])
```

Comment:
```suggestion
                metadata_required_column.set_options([])
```

Würde ich auch eher leer lassen